source/connect.txt

```diff
@@ -122,6 +122,19 @@ option for programmatic usage of ``mongosh``, like a :driver:`driver
    - To provision access to a MongoDB deployment, see :manual:`Database
      Users </core/security-users/>`.
 
+Connect with OpenID Connect
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
```

Comment:
(suggestion)
Consider linking over to the OIDC concept page in the Server docs. It might be good to give a newer user some additional context.